Change some grammar in README (#127)
nuxt-community/sitemap-module: README.md

- Default: `1000 * 60 * 15` (15 Minutes)

Defines how frequently sitemap **routes** should be updated (value in milliseconds).

Please note that after each invalidation, `routes` will be evaluated again (see [routes declaration](#routes-declaration) section).

> This option is only available for nuxt in its **ssr** mode.

Reviewer: ```suggestion
This option is only available on **ssr** mode.
```

Author: Indeed it is probably not necessary to repeat nuxt everywhere. However in this case I would replace "on ssr mode" with "in ssr mode"

Reviewer: ```suggestion
This option is only available in **ssr** mode.
```
